SDK/AppCenterData/Microsoft.AppCenter.Data.iOS/Data.cs

```diff
@@ -20,6 +21,25 @@ public partial class Data : AppCenterService
         /// </value>
         [Preserve]
         public static Type BindingType => typeof(MSData);
+        static RemoteOperationDelegate _remoteOperationDelegate = new RemoteOperationDelegate();
+
+        static Data()
+        {
+            _remoteOperationDelegate.DataDidCompleteRemoteOperation = (operation, documentMetadata, error) =>
+            {
+                if (RemoteOperationCompleted == null)
+                {
+                    return;
+                }
+                RemoteOperationCompleted(typeof(Data), new RemoteOperationCompletedEventArgs
```

Comment:
Is there a possible race condition here? Maybe the `RemoteOperationCompleted` callback can be made `null` before it is invoked but after checking. Maybe save it to a local variable and then check for null/invoke?

Reply:
Or just use `RemoteOperationCompleted?.Invoke()`